core/src/Cuda/Kokkos_Cuda_Half.hpp

```diff
@@ -45,6 +45,7 @@
 #ifndef KOKKOS_CUDA_HALF_HPP_
 #define KOKKOS_CUDA_HALF_HPP_
 
+#include <iostream>
```

Comment:
question: can we somehow avoid including this? For example by just including forward declarators, and then implement the function in a cpp file? I would like us medium term to reduce the amount of standard library stuff we include for every user to help with compile times.

Reply:
yea, we can just remove support for the io operators.

Reply:
resolved via force push to 86edddb.